samples/msal-node-samples/client-credentials-with-cert-from-key-vault/jest.config.cjs

```diff
@@ -6,4 +6,5 @@
 module.exports = {
     displayName: "Client Credentials with Certificate from Key Vault",
     preset: "../../e2eTestUtils/jest-puppeteer-utils/jest-preset-no-setup.js",
+    extensionsToTreatAsEsm: [".ts", ".tsx"],
```

Comment:
Why is this needed?

Reply:
It's not, I deleted it

Reply:
I was getting some compilation errors without this

Reply:
https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=1385457&view=logs&j=e2a201cd-fff7-5f83-ed5f-a5e7e4e44190&t=b5e84f64-0e7c-5c3b-7eb2-5014792c7f1e&l=80

Reply:
Is this an old run? It's referencing `jest.config.js` being treated as ESM but the file is, correctly, named `jest.config.cjs` now

Reply:
It is an old run. I receive that error when the file ends with ".js" instead of ".cjs". I can Google the difference I suppose, but for the purpose of discussion here, what is the ;TLDR difference between the two?

I can try keeping the file as ".cjs" and removing that line you asked the question about, and see if I still receive the build error.

Reply:
If you've specified `type: "module"` in the package.json, `.js` file extensions are treated as ESM. Without the `type` specified `.js` extensions are treated as CJS. To explicitly use CJS or ESM use `.cjs` or `.mjs` respectively. In this case the config is written using CJS syntax so it either needs to use `.cjs` or remove `type: "module"` from the package.json